src/ensembl/src/content/app/entity-viewer/shared/rest/rest-data-fetchers/proteinData.ts

```diff
@@ -14,12 +14,14 @@
  * limitations under the License.
  */
 
-import { restProteinStatsAdaptor } from '../rest-adaptors/rest-protein-adaptor';
+import { restProteinSummaryAdaptor } from '../rest-adaptors/rest-protein-adaptor';
 import { TranscriptInResponse } from './transcriptData';
 
-export type XrefsInResponse = {
+export type Xrefs = {
```

Comment:
Is it the type of a single reference? Should it be called Xref (or CrossReference? or ExternalReference?) then?

Reply:
Yes that's right. I should make this singular.